src/processorgen/specs/error.json

```
@@ -0,0 +1,45 @@
+{
+  "specification": {
+    "name": "error",
+    "summary": "Returns an error for all records that get passed to the processor.",
+    "description": "Any time a record is passed to this processor it returns an error,\nwhich results in the record being sent to the DLQ if it's configured, or the pipeline stopping.\n\n**Important:** Make sure to add a [condition](https://conduit.io/docs/processors/conditions)\nto this processor, otherwise all records will provoke an error.",
```

Only leaving this suggestion since I left it on the source as well https://github.com/ConduitIO/conduit/pull/1598.

```suggestion
    "description": "Any time a record is passed to this processor it returns an error,\nwhich results in the record being sent to the DLQ if it's configured, or the pipeline stopping.\n\n**Important:** Make sure to add a [condition](https://conduit.io/docs/processors/conditions)\nto this processor, otherwise all records will trigger an error.",
```